tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/GenerateContent.java

```diff
@@ -213,13 +213,13 @@ public void execute() throws BaseNessieClientServerException {
             api.commitMultipleOperations()
                 .branch(commitToBranch)
                 .commitMeta(
-                    CommitMeta.builder()
-                        .message(
-                            String.format(
-                                "Commit #%d of %d on %s", commitNum, numCommits, branchName))
-                        .author(System.getProperty("user.name"))
-                        .authorTime(Instant.now())
-                        .build());
+                    withStandardCommitMetaFields(
+                        CommitMeta.builder()
+                            .message(
+                                String.format(
+                                    "Commit #%d of %d on %s", commitNum, numCommits, branchName))
+                            .authorTime(Instant.now())
```

Comment:
Think it's fine to remove authorTime here, because it'll be set on the server side anyways.
This would make the `withStandardCommitMetaFields` take just a `String` for the message, saving one builder-from-build round trip - and makes the code easier.

Reply:
updated